protostar/compiler/project_cairo_path_builder.py

```diff
@@ -1,7 +1,14 @@
 from pathlib import Path
 
 
-class ProjectCairoPathBuilder:
+class LinkedLibrariesBuilder:
```

Comment:
[nitpick]: This is kind of confusing since linked libraries is a term for cairo1 exclusively, and this class is a required interface in CLI class. If we extracted the interface, named it like `CairoPathGetter` or `CompilationPathBuilder` and inherited in both classes, we could require the interface in CLI class. As per the name of the method (`build_project_cairo_path_list`): this is a method which is implemented in the `LikedLibrariesBuilder` and is kind of mixed into the cairo1 context where the cairo path doesn't really exist anymore. 

Now that i'm thinking after writing this, perhaps we should divide the 2 concepts from each other, since we changed argument name anyway? Even if we have to add an extra argument to the legacy command.

I understand that it's a workaround, but it's not ideal nonetheless. I'm marking it as nitpick, to be addressed now or later if we don't have time for it rn.

Reply:
#1559

Reply:
I will address this but I want first to have cairo0 runner removed